lib/chronic_duration.rb

```diff
@@ -47,7 +47,7 @@ def output(seconds, opts = {})
     month = 22 * day
     year = 260 * day
 
-    if seconds >= 31557600 && seconds%year < seconds%month
+    if seconds >= year && !opts[:limit_to_hours]
```

Comment:
Not sure how this ever worked before. 

- The `31557600` refers to a full year not a working year. 
- `seconds%year < seconds%month` just doesn't make sense to me

Reply:
I have no idea what `seconds%year < seconds%month` means!